akka-diagnostics/src/test/scala/akka/diagnostics/ConfigCheckerSpec.scala

```diff
@@ -74,6 +74,18 @@ class ConfigCheckerSpec extends AkkaSpec {
 
   "The ConfigChecker" must {
 
+    "find no warnings in PersistenceTestKitPlugin.scala configuration" in {
+      val extendedReference = ConfigFactory.load("persistence.conf").withFallback(reference)
+      val c = ConfigFactory
+        .parseString("""
+        akka.persistence.testkit.snapshotstore.pluginid.snapshot-is-optional = false
+        """)
```

Comment:
For more test coverage you can add the persistence-testkit as a test dependency and then include all of them
```
PersistenceTestKitPlugin.config
  .withFallback(PersistenceTestKitSnapshotPlugin.config)
  .withFallback(PersistenceTestKitDurableStateStorePlugin.config)
  .withFallback(ConfigFactory
         .parseString("""
         akka.persistence.testkit.snapshotstore.pluginid.snapshot-is-optional = false
         """)
  .withFallback(reference)
```

Reply:
I created the test and of course passing but you wouldn't believe what this triggers.
When importing `"com.typesafe.akka" %% "akka-persistence-testkit" % AkkaVersion` the test `must log warning when ActorSystem startup` fails???
I'm trying to figure out how is that even possible

Reply:
It seems that the problem is that we have now two ActorSystems, one with the offending configuration and one without it.

Reply:
Patrik solved it. Thanks!